File: Examen1Bimestre/app/src/main/java/com/example/examen1bimestre/Cancion.kt
package com.example.examen1bimestre

class Cancion(
    var id: Int,
    var createdAt: Long,
    var updatedAt: Long,
    var nombre:String,
    var autor:String,
    var genero:String,
    var acordes:String) {

    override fun toString(): String {
        return "Nombre: ${nombre}, Autor:${autor}"
    }
}

File: deber-crud-sails/api/models/Cancion.js
/**
 * Cancion.js
 *
 * @description :: A model definition represents a database table/collection.
 * @docs        :: https://sailsjs.com/docs/concepts/models-and-orm/models
 */

module.exports = {

  attributes: {

  //var nombre:String, var autor:String,var genero:String,var acordes:String
  nombre:{
    type:'string',
    required:true,
  },
  autor:{
    type:'string',
  },
  
  genero:{
    type:'string',

  },
  acordes:{
    type:'string'
  }


  },

};

